### PR TITLE
Rename File drop to Secure drop

### PR DESCRIPTION
**Pull request by oparoz**

### CODE REVIEW

File: core/js/sharedialoglinkshareview.js [hunk 1]
```diff
@@ -437,7 +437,7 @@
 				publicEditing: publicEditable,
 				publicEditingChecked: publicEditingChecked,
 				publicEditingLabel: t('core', 'Allow editing'),
-				hideFileListLabel: t('core', 'File drop (upload only)'),
+				hideFileListLabel: t('core', 'Secure drop (upload only)'),
```
Comment (LukasReschke) [inline core/js/sharedialoglinkshareview.js]: Should "Secure drop" here really be translateable? I think for branding purposes we should make this stay consistent?

**Review by LukasReschke — DISMISSED** [review]

File: core/js/sharedialoglinkshareview.js (hunk 1 above)
Comment (oparoz) [inline core/js/sharedialoglinkshareview.js]: Good point

Comment (MorrisJobke) [inline core/js/sharedialoglinkshareview.js]: Fixed

**Review by MorrisJobke — APPROVED** [review]

**Review by rullzer — APPROVED** [review]
Fine by me

**Pull request merged**